docs/en/writing-running-appium/android_coverage.md

```diff
@@ -0,0 +1,292 @@
+# Android Coverage Guide
+## Requirement
+* Only Emulators or rooted phones are acceptable
+* Need add UiAutomator2 supprt in your apps. That is to say, you need implement a subclass of `Instrumentation`. `Instrumentation` will be responsible to collect your coverage. 
+* Need to implement a BroadcastReceiver to export coverage to disk files. This is because `Instrumentation` only collects data into memory.
+
+## Project Structure
+* You may need the following structure of files:
+```
+src/main/java/com/example/pkg
```

Comment:
For the sake of consistency, all of the descriptions in this file structure ought to begin and end with a `*`, or none of them should. Right now it's a mixture.